chia/wallet/trading/offer.py

```diff
@@ -179,13 +179,8 @@ def get_pending_amounts(self) -> Dict[str, int]:
             pending_dict[name] = 0
             for coin in coins:
                 root_removal: Coin = coin
```

Comment:
Sure, my instinct was not to assign to the thing `in coins` but if that doesn't matter then I'm game.

Reply:
I'm going to resolve these conversations because I reworked the function in https://github.com/Chia-Network/chia-blockchain/pull/9541 and took these comments into account.